torch/csrc/inductor/aoti_runner/model_container_runner.cpp

```diff
@@ -46,6 +47,18 @@ AOTIModelContainerRunner::AOTIModelContainerRunner(
   get_call_spec_func_ = reinterpret_cast<decltype(get_call_spec_func_)>(
       model_so_->sym("AOTInductorModelContainerGetCallSpec"));
 
+  // Hack to find the json file name from the model so file
+  size_t lastindex = model_so_path.find_last_of(".");
+  const std::string& json_filename =
```

Comment:
this only happens to work because of an arcane C++ quirk (temporary lifetime extension -- https://abseil.io/tips/107); `std::string json_filename = ...` would be clearer